### PR TITLE
Parsing sec files

### PR DESCRIPTION
**Pull request by Skydivizer**
Most logic is done. To finish reading completely, only the rough format of objects is needed.

Still need to figure out the internal formats as well, (just stored as raw data):
lights,
tiles,
roofs,
scripts + script tiles
etc.

### CODE REVIEW

**Comment by LeonCloudEndure** [comment]
This is amazing :)
BTW i never did anything with it, but some cool guy gave me access to his repo where he done the same work in pascal, i can ask him if i can share the code with you (or he can directly) so that you can use his research (if you don't mind reading pascal, i think it was pascal, maybe i misremembered and it was c)

**Comment by Skydivizer** [comment]
That would be awesome. Any research would probably help me immensely.

File: formats/obj.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    )
+
+    # Size in bytes of included field flags.
+    # len(Fields.type) / 32
+    type_flags_length = (
+        3, # 00 Wall
+        3, # 01 Portal
+        3, # 02 Container
+        3, # 03 Scenery
+        0, # 04 Projectile
```
Comment (iamkisly) [inline formats/obj.py:71]: Длина флагов заголовка для Projectile 3*32. Подробнее можно посмотреть тут : https://github.com/iamkisly/ArcanumFileFormats/wiki/Example-0x04-:-Projectile

Comment (Skydivizer) [reply formats/obj.py:71]: Thanks for the information!

**Pull request merged**